swagger/body-complex.json

```diff
@@ -1179,7 +1179,8 @@
           "enum": [ "cyan", "Magenta", "YELLOW", "blacK" ],
           "x-ms-enum": {
             "name": "CMYKColors",
-            "modelAsString": true
+            "modelAsString": true,
```

Comment:
I'm just thinking what we're calling this  -- I want to go and talk with a few folks so we have everything exactly right